static/app/views/starfish/utils/retryHandlers.tsx

```diff
@@ -0,0 +1,7 @@
+export function shouldRetryHandler(failureCount: number, error: Error) {
+  return failureCount < 3 && error.message.includes('429');
+}
```

Comment:
Nice! This is good and generic, could be called something helpful like `retryOnThrottleResponse` or something similar!